index.html

```diff
@@ -104,7 +103,7 @@ <h1 id="homeHeading">Increasing the number of talented Software Engineers in Lat
                 <div class="col-lg-8 col-lg-offset-2 text-center">
                     <h2 class="section-heading">What is omegaUp?</h2>
                     <hr class="light">
-                    <p class="text-faded">omegaUp is a non-profit organization (501c3) aimed to increase the number of talented Software Engineers in Latin America. Students can use our open source platform <mark><a href="https://omegaup.com" target="_blank">omegaUp.com</a></mark> to learn and improve their Computer Science skills through coding challenges with a fun and competitive approach. </p>
+                    <p class="text-faded">omegaUp is a non-profit organization (501c3) aimed to increase the number of talented Software Engineers in Latin America. Students can use our open source platform <mark><a href="https://omegaup.com" target="_blank">omegaUp.com</a></mark> to learn and improve their software engineering skills by solving coding challenges and competing with other peers in programming contests organized by the community.</p>
```

Comment:
```suggestion
                    <p class="text-faded">omegaUp is a non-profit organization (501c3) aimed to increase the number of talented Software Engineers in Latin America. Students can use our open source platform <mark><a href="https://omegaup.com" target="_blank">omegaUp.com</a></mark> to learn and improve their software engineering skills by solving coding challenges and competing with their peers in programming contests organized by the community.</p>
```

"other peers" no tiene mucho sentido, porque peer es algo relativo ("whose peer?")